Exploration Queries/IoT/Process_byIoTDevice.yaml

```diff
@@ -0,0 +1,24 @@
+﻿Id: 1f3ecde7-5c69-4d44-ac93-5feac6d1cd2f
+DisplayName: Most frequent command executions on the device
+Description: Most frequent command executions on the device
+InputEntityType: IoTDevice
+InputFields:
+  - DeviceId
+  - IoTHub
+OutputEntityTypes:
+  - Process
+QueryPeriodBefore: 30m
+QueryPeriodAfter: 30m
+DataSources: SecurityIoTRawEvent
+query: |
+
+  let Process_byIoTDevice = (v_IotDevice_DeviceId:string, v_IoTDevice_IoTHub:string){
+  SecurityIoTRawEvent 
+  | where RawEventName == 'ProcessCreate'
+  | where AssociatedResourceId == parse_json(v_IoTDevice_IoTHub)['ResourceId'] and DeviceId == v_IotDevice_DeviceId
+  | extend Process_CommandLine = tostring(parse_json(EventDetails)['CommandLine'])
+  | summarize procCount = count() by Process_CommandLine
```

Comment:
We are only producing the Command line, I think it is important to include other information in a packed field.  Similar to what is done here - https://github.com/Azure/Azure-Sentinel/blob/master/Exploration%20Queries/SecurityEvent/WinHosts_WithThisProcess.yaml

Reply:
I'll see what I can add, but we don't have much more info from IoT

Reply:
Okay, if there is no other context that would be useful to an investigation, then I am good here.  I am not an expert on IoT logs, so if this is all all the context we can provide we go with it. :)